torch/fx/experimental/symbolic_shapes.py

```diff
@@ -287,6 +286,8 @@ def check_supported_type(x):
                 return sympy.floor(base)
             if isinstance(base, sympy.Integer) and isinstance(divisor, sympy.Integer):
                 return base // divisor
+            if isinstance(base, (sympy.Integer, sympy.Float)) and isinstance(divisor, (sympy.Integer, sympy.Float)):
+                return sympy.floor(base / divisor)
```

Comment:
This is a separate case to avoid calling floor when both args are Integers.

Reply:
Okay, I'll merge these two PRs after review as is to avoid stressing the CI for no good reason, then will fix this in a follow up.